src/install/index.js

```diff
@@ -51,8 +51,16 @@ const findDownloadedBuild = deployDoc => {
 };
 
 const downloadBuild = deployDoc => {
-  debug(`Downloading ${keyFromDeployDoc(deployDoc)}, this may take some time…`);
-  return DB.builds.get(keyFromDeployDoc(deployDoc), { attachments: true, binary: true })
+  const docKey = keyFromDeployDoc(deployDoc);
+  debug(`Downloading ${docKey}, this may take some time…`);
+  return DB.builds.get(docKey, { attachments: true, binary: true })
+    .catch(err => {
+      if (err && err.status === 404) {
```

Comment:
I wonder if this check should less specific. Essentially what we're testing for is, should we retry this upgrade or not. Any HTTP status code in the 4xx range indicates the fault is with the client, in which case retrying is likely going to fail. What about...

```suggestion
      if (err && err.status && err.status >= 400 && err.status < 500) {
```